src/binder/assemblybinder.cpp

```diff
@@ -1425,23 +1423,19 @@ namespace BINDER_SPACE
                             {
                                 fileName.Append(W(".ni.dll"));
                                 hr = GetAssembly(fileName,
-                                    fInspectionOnly,
-                                    FALSE, /* fIsInGAC */
-                                    TRUE /* fExplicitBindToNativeImage */,
-                                    &pAssembly);
+                                                 fInspectionOnly,
+                                                 FALSE, // fIsInGAC
```

Comment:
I don't know what's the usual guideline here, but, unless a boolean parameter to a function has very obvious meaning (e.g. `Widget::SetIsVisible(bool)`), it's a good idea to pass enum types as parameters, even if they have only two possible options.  It's a lot clearer from the call site what's happening, without requiring comments (you also get help from the compiler if you pass a value that doesn't apply to that particular enum, which is nice).

Reply:
I agree to a point:
* declaring the enum adds non-trivial cost in terms of code lines and so on... and always makes me wonder if it has some hidden meaning
* it's just not how the code is written in the runtime - which doesn't mean we should not look at changing it, just that we would be introducing something relatively "new" and potentially confusing

Reply:
In this particular case - It is likely that most of these flags can be just deleted after further analysis, e.g. there is no GAC concept in CoreCLR, why do we have a flag for it?

Reply:
I've looked into that a little bit - we're using it as "IsTPAAssembly" basically - and there's a piece of logic that depends on it: https://github.com/dotnet/coreclr/blob/54d9d9b2698c9628f43494e46f855996b31838bd/src/binder/clrprivbindercoreclr.cpp#L160

Maybe there's a way to remove that as well... but for this change I just didn't want to go into all these details... yet.